### DISCUSSION TITLE
Extension stopped functioning - maybe a change to Microsoft API?

### DISCUSSION THREAD

**Discussion by Conzar** [jqnatividad/ckanext-officedocs#22] [Q&A]
The extension was working without issue until recently.  Now the preview shows the following:

```
An error occurred
We're sorry, but for some reason we can't open this for you.
Learn more
```

The learn more is a hyperlink that redirects with the following error:

```
This site can’t be reachedThe webpage at https://www.office.com/redir.aspx?assetid=HA102724036 might be temporarily down or it may have moved permanently to a new web address.
ERR_INVALID_RESPONSE
```

This is not working on ckan version 2.7 (it was working before on 2.7).

**Comment by jqnatividad** [comment]
Hi @Conzar ,
Is this still a problem? If so, you may want to check out https://github.com/ckan/ckan/issues/6256
